suites/squid/common/regression/rados/rados_4-node-ecpools.yaml

```diff
@@ -0,0 +1,738 @@
+            pool-1:
+              profile_name: ec86_10
+              pool_name: ec86_pool10
+              pool_type: erasure
+              pg_num: 32
+              k: 8
+              m: 6
+              create_rule: false
+              crush-osds-per-failure-domain: 4
+              crush-num-failure-domains: 4
+              plugin: jerasure
+              crush-failure-domain: host
+              target_size_ratio: 0.8
+              rados_write_duration: 50
+              rados_read_duration: 50
+              delete_pool: true
+      desc: Specifying pool bounds on pool Pgs - Verify target_size_ratio
+
+  - test:
+      name: Autoscaler test - EC 8+6 pool pg_num_min
+      module: pool_tests.py
+      polarion-id: CEPH-83573425
+      config:
+        verify_pg_num_min:
+          configurations:
+            pool-1:
+              profile_name: ec86_11
+              pool_name: ec86_pool11
+              pool_type: erasure
+              k: 8
+              m: 6
+              create_rule: false
+              crush-osds-per-failure-domain: 4
+              crush-num-failure-domains: 4
+              plugin: jerasure
+              crush-failure-domain: host
+              pg_num_min: 16
+              rados_write_duration: 50
+              rados_read_duration: 50
+              delete_pool: true
+      desc: Specifying pool bounds on pool Pgs - Verify pg_num_min
+
+  - test:
+      name: client pg access
+      module: test_client_pg_access.py
+      polarion-id: CEPH-83571713
+      config:
+        verify_client_pg_access:
+          num_snapshots: 20
+          configurations:
+            pool-1:
+              profile_name: ec86_12
+              pool_name: ec86_pool12
+              pool_type: erasure
+              k: 8
+              m: 6
+              create_rule: false
+              crush-osds-per-failure-domain: 4
+              crush-num-failure-domains: 4
+              plugin: jerasure
+              crush-failure-domain: host
+              disable_pg_autoscale: true
+      desc: many clients clients accessing same PG with bluestore as backend
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: replicated
+        pool-2-type: erasure
+        pool-1-conf: sample-pool-1
+        pool-2-conf: sample-pool-3
+        pool_configs_path: "conf/squid/rados/test-confs/pool-configurations.yaml"
+      desc: Migrating data between different pools. Scenario-2. RE -> EC
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: erasure
+        pool-2-type: replicated
+        pool-1-conf: sample-pool-3
+        pool-2-conf: sample-pool-3
+        pool_configs_path: "conf/squid/rados/test-confs/pool-configurations.yaml"
+      desc: Migrating data between different pools. Scenario-3. EC -> RE
+
+  - test:
```

Comment:
make sure the pool configs shared are valid 4 node pools

Reply:
Done. All the pool configs used in this test suite are either for 2+2 or 8+6 EC config